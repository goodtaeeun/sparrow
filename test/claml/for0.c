int main(int argc, char *argv[]) {
  int i;
  for (i = 0; argv[i]; i++)
    ;
  sparrow_print(i);
}
