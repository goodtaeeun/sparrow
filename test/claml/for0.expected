/* Generated by CIL v. 1.7.3 */
/* print_CIL_Input is false */

#line 1 "for0.c"
int main(int argc, char *argv[]) {
  int i;
  int __cil_tmp4;

  {
#line 3
    i = 0;
    {
#line 3
      while (1) {
      while_continue: /* CIL Label */;
#line 3
        if (!argv[i]) {
#line 3
          goto while_break;
        }

#line 3
        i++;
      }
    while_break: /* CIL Label */;
    }
    {
#line 5
      sparrow_print(i);
    }
    return (0);
  }
}
